src/test/java/com/homework/lesson09/MyArrayListTest.java

```diff
@@ -0,0 +1,58 @@
+package com.homework.lesson09;
+
+import org.junit.Test;
+import org.junit.Before;
+
+import static org.junit.Assert.assertEquals;
+
+public class MyArrayListTest {
+    private MyArrayList<Integer> myArrayList;
+
+    @Before
+    public void setUp() {
+        myArrayList = new MyArrayList<>();
+        myArrayList.add(1);
+    }
+
+    @Test
+    public void get() {
+        int expected = myArrayList.get(0);
+        int actual = 1;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void add() {
+        myArrayList.add(2);
+        int expected = myArrayList.get(1);
```

Comment:
what you have in your list is actual value
and what you need to get is expected value